src/Razor/src/rzls/rzls.csproj

```diff
@@ -52,7 +52,9 @@
   <Target Name="PublishAllRids">
     <ItemGroup>
       <!-- Transform RuntimeIdentifiers property to item -->
-      <RuntimeIdentifierForPublish Include="$(RuntimeIdentifiers)" />
```

Comment:
Is `RuntimeIdentifiers` still used? if so, should it be updated with conditions, instead of directly updating `RuntimeIdentifiersForPublish`?

Reply:
Good idea - updated!